test/releases/releases.go

```diff
@@ -28,14 +30,45 @@ type Release struct {
 
 // Releases are populated by respective release modules using init()
 var releases = []Release{}
-var releasesSorted = false
+var releasesProcessed = false
 
 // Returns list of releases sorted from oldest to newest
 func Releases() []Release {
-	if releasesSorted {
+	if releasesProcessed {
 		return releases
 	}
 
+	// Keep only releases matching the selector
+	if releasesSelectorRaw, found := os.LookupEnv("RELEASES_SELECTOR"); found {
```

Comment:
What happens if we have both selector and deselector?

Reply:
Selectors are applied first and then we use deselectors on that. In other words, they are merged. You cannot SELECT something that was DESELCTED.